crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -79,6 +84,27 @@ impl CachedComputePipelineId {
     }
 }
 
+/// Index of a cached pipeline of any kind in a [`PipelineCache`]. This is essentially a union of
+/// all pipeline ID types.
+#[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
+pub enum CachedAnyPipelineId {
+    RenderPipeline(CachedRenderPipelineId),
+    ComputePipeline(CachedComputePipelineId),
+}
+
+impl CachedAnyPipelineId {
```

Comment:
I think the name `AnyCachedPipelineId` makes more sense here. I'm not a native speaker btw.

Reply:
Done. Honestly I was having a hard time deciding between the two :P I initially decided to go with `CachedAnyPipelineId` to match the `CachedComputePipelineId` and `CachedRenderPipelineId`, but I am personally fine with `AnyCachedPipelineId`.